assets/components/leftMarginSection/leftMarginSection.scss

```diff
@@ -1,6 +1,13 @@
 .component-left-margin-section {
   display: flex;
 
+  &.component-left-margin-section--blue {
```

Comment:
Because this class is unique across the codebase I think you could put this rule-set at the top level of this file?